### PR TITLE
feat: disable months in the past

### PR DESCRIPTION
**Pull request by christianrank**

### CODE REVIEW

**Review by wwwbruno — APPROVED** [review]

**Comment by emrox** [comment]
you forgot to add the new props to the `propTypes`

**Comment by emrox** [comment]
adding some tests for the new props would be good

File: tests/Calendar.spec.jsx [hunk 1]
```diff
@@ -28,6 +28,18 @@ describe('Calendar', () => {
       const wrapper = mount(<Calendar showToday={false}/>);
       expect(wrapper.find('.rc-calendar-today-btn').length).toBe(0);
     });
+
+    it('render disableMonthsInPast true correctly', () => {
```
Comment (emrox) [inline tests/Calendar.spec.jsx]: you could group them within an `describe` block for better readability

File: tests/Calendar.spec.jsx [hunk 2]
```diff
@@ -28,6 +28,18 @@ describe('Calendar', () => {
...
+      const wrapper = mount(<Calendar disableMonthsInPast />);
+      expect(wrapper.find('.rc-calendar-prev-year-btn.disabled').length).toBe(1);
```
Comment (emrox) [inline tests/Calendar.spec.jsx]: there is the `.toHaveLength(number)` matcher you could use
https://facebook.github.io/jest/docs/en/expect.html

**Review by emrox — CHANGES_REQUESTED** [review]

File: tests/Calendar.spec.jsx (hunk 1 above)
Comment (christianrank) [reply tests/Calendar.spec.jsx]: cfd5a2f

File: tests/Calendar.spec.jsx (hunk 2 above)
Comment (christianrank) [reply tests/Calendar.spec.jsx]: cfd5a2f

Comment (emrox) [reply tests/Calendar.spec.jsx]: 👍🏻

**Review by emrox — APPROVED** [review]

**Pull request merged**